contributed_definitions/nyaml/NXmpes_xps_liquid.yaml

```diff
@@ -0,0 +1,314 @@
+                "Selective area",
+                "Deflector",
+                "PEEM",
+                "Momentum Microscope",
+              ]
+          mode(NX_CHAR):
+            exists: recommended
+            doc: "Labelling of the lens setting in use."
+          projection(NX_CHAR):
+            exists: recommended
+            doc: "The space projected in the angularly dispersive directions, i.e. real or reciprocal."
+            enumeration: ["real", "reciprocal"]
+        ANALYSER_DISPERSIVE_ELEMENT(NXenergydispersion):
+          scheme(NX_CHAR):
+            enumeration:
+              [
+                "tof",
+                "hemispherical",
+                "double hemispherical",
+                "cylindrical mirror",
+                "display mirror",
+                "retarding grid",
+              ]
+          pass_energy(NX_FLOAT):
+            unit: NX_ENERGY
+            doc: "Energy of the electrons on the mean path of the analyser. Pass energy for hemispherics, drift energy for tofs."
+          energy_scan_mode(NX_CHAR):
+            doc: "Way of scanning the energy axis (fixed or sweep)."
+            enumeration: ["fixed", "sweep"]
+          ENTRANCE_SLIT(NXaperture):
+            exists: recommended
+            doc: "Aperture generating the momentum and/or energy filtering."
+            description(NX_CHAR):
+              doc: "Type of aperture inserted in the beam."
+              enumeration: ["slit", "pinhole", "iris"]
+            shape(NX_CHAR):
+              doc: "Description of the shape of the active part of the aperture, curved or straight for horizontal slits, square or round for pinhole etc."
+              enumeration:
+                [
+                  "curved",
+                  "straight",
+                  "circle",
+                  "square",
+                  "hexagon",
+                  "octagon",
+                  "bladed",
+                ]
+              pass_energy(NX_FLOAT):
+                unit: NX_LENGTH
+                doc: "The relevant dimension for the aperture (slit width, pinhole diameter etc)."
+        ELECTRON_DETECTOR(NXdetector):
+          amplifier_type(NX_CHAR):
+            exists: recommended
+            doc: "Type of electron amplifier in the first amplification step."
+            enumeration: ["MCP", "channeltron"]
+          detector_type(NX_CHAR):
+            doc: "Description of the detector type."
+            enumeration:
+              [
+                "DLD",
+                "Phosphor+CCD",
+                "Phosphor+CMOS",
+                "ECMOS",
+                "Anode",
+                "Multi-anode",
+              ]
+          DATA(NXdata): # Raw signal without calibrated axes.
+            exists: recommended
+            \@signal(NX_CHAR):
+              enumeration: ["raw"]
+            raw(NX_NUMBER): # There is a block of numbers named raw.
+              doc: "Raw data before calibration."
+        GEOMETRY(NXtransformations):
```

Comment:
The general idea of NXtransformations is different. It is intended as transformation matrices attached to an instrument part, i.e., the analyzer would have a transformation matrix attached to transform a coordinate system into its frame.